bin/setup

```diff
@@ -16,16 +15,13 @@ FileUtils.chdir APP_ROOT do
   # Add necessary setup steps to this file.
 
   puts "== Installing dependencies =="
-  system! "gem install bundler --conservative"
```

Comment:
removed in https://github.com/rails/rails/commit/37f4073277958072ef9069ddcc4e376a4116c7e4